integration-tests/deployment/clo/offchain_client_impl.go

```diff
@@ -0,0 +1,213 @@
+package clo
```

Comment:
forked from clo repo until KS-455 can be tackled https://github.com/smartcontractkit/feeds-manager/tree/develop/api/models